examples/tcp_client/tcp_client.cc

```diff
@@ -0,0 +1,157 @@
+/**************************************************************************/ /**
```

Comment:
the name of the file should reflect this is Linux only (or we should consider using an OS independent library like boost sockets)

Reply:
Ok, I'll rename. I specifically wanted to avoid bring in Boost since it is a bear of a dependency, and Boost ASIO is far from easy to understand.